formats/json-tests/commonTest/src/kotlinx/serialization/SerializersLookupTest.kt

```diff
@@ -279,6 +279,13 @@ class SerializersLookupTest : JsonTestBase() {
         }
     }
 
+// TODO uncomment when Kotlin 1.7.20 is released
```

Comment:
Should this be uncommented?

Reply:
Not actual code

Reply:
This change wasn't include in the merged commit.

Reply:
https://github.com/Kotlin/kotlinx.serialization/pull/1991/commits/a5d92ec747290199d7828883bd71b6b4d107e6ab